runtimes/core/src/log/logger.rs

```diff
@@ -202,6 +212,8 @@ impl Logger {
             }
         }
 
+        self.write_to_trace(request, level, &msg, &values);
```

Comment:
We're inserting the message as a value above. Do we need to pass it in here, too? It requires cloning the message, per above.